.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   build-and-test:
     working_directory: ~/prontolista/api
     docker:
-      - image: circleci/python:3.7.4
+      - image: circleci/python:3.7.4-node-browsers
```

Comment:
Why we need to use -node-browser arr?

Reply:
It's because we need to run Cypress ja. The `circleci/python:3.7.4-node-browsers` already has the Yarn and NPM installed for us. 🤟🏻

Reply:
orrrrrrr๋